### PR TITLE
add Obtainium and 2FAS

### PR DESCRIPTION
**Pull request by htetoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -508,6 +508,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Password & Authentication
 
+* [**2FAS**](https://github.com/twofas/2fas-android)
```
Comment (offa) [inline README.md:511]: The app includes proprietary google services.

**Review by offa — CHANGES_REQUESTED** [review]
Thanks for your contribution. 2FAS seems to include proprietary components, as far as I can tell.

File: README.md [hunk 2]
```diff
@@ -813,6 +814,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**APKLab**](https://github.com/APKLab/APKLab)
 * [**Apktool**](https://github.com/iBotPeaches/Apktool)
 * [**APK Editor Studio**](https://github.com/kefir500/apk-editor-studio)
+* [**Obtainium**](https://github.com/ImranR98/Obtainium)
```
Comment (offa) [inline README.md:817]: Could fit under *App Store* too? :-)

**Review by htetoh — COMMENTED** [review]
Done.

**Comment by offa** [comment]
Perfect, thank you! :-)

**Review by offa — APPROVED** [review]

**Pull request merged**